app/src/pages/ExploreDataPage.tsx

```diff
@@ -218,48 +148,78 @@ function CarouselMadLib(props: {
   madLib: MadLib;
   setMadLib: (updatedMadLib: MadLib) => void;
 }) {
+  function updateMadLib(phraseSegementIndex: number, newValue: string) {
+    let updatePhraseSelections: PhraseSelections = {
+      ...props.madLib.activeSelections,
+    };
+    updatePhraseSelections[phraseSegementIndex] = newValue;
+    props.setMadLib({
+      ...props.madLib,
+      activeSelections: updatePhraseSelections,
+    });
+  }
+
   return (
-    <React.Fragment>
+    <Grid container spacing={1} justify="center" style={{ lineHeight: "50pt" }}>
       {props.madLib.phrase.map(
         (phraseSegment: PhraseSegment, index: number) => (
           <React.Fragment key={index}>
             {typeof phraseSegment === "string" ? (
-              <React.Fragment>{phraseSegment}</React.Fragment>
+              <Grid item>{phraseSegment}</Grid>
             ) : (
-              <FormControl>
-                <Select
-                  className={styles.MadLibSelect}
-                  name={index.toString()}
-                  defaultValue={props.madLib.defaultSelections[index]}
-                  value={props.madLib.activeSelections[index]}
-                  onChange={(event) => {
-                    let phraseIndex: number = Number(event.target.name);
-                    let updatePhraseSelections: PhraseSelections = {
-                      ...props.madLib.activeSelections,
-                    };
-                    updatePhraseSelections[phraseIndex] = event.target
-                      .value as string;
-                    props.setMadLib({
-                      ...props.madLib,
-                      activeSelections: updatePhraseSelections,
-                    });
-                  }}
-                >
-                  {Object.entries(phraseSegment)
-                    .sort((a, b) => a[0].localeCompare(b[0]))
-                    .map(([key, value]) => (
-                      // TODO - we may want to not have this alphabetized by ID by default
-                      <MenuItem key={value} value={key}>
-                        {value}
-                      </MenuItem>
-                    ))}
-                </Select>
-              </FormControl>
+              <>
+                {/* TODO - don't use this hack to figure out if its a FIPS or not*/}
```

Comment:
yikes... this one may be worth fixing sooner rather than later

Reply:
Yes definitely... I had something out to address it but I think doing it properly may be a larger refactor so I was gonna wait till everything else got checked in...

Reply:
Sounds good, this is big enough as is :P